src/Group.php

```diff
@@ -1155,5 +1155,8 @@ public static function updateLastGroupChange()
     {
         global $GLPI_CACHE;
         $GLPI_CACHE->set('last_group_change', $_SESSION['glpi_currenttime']);
+
+        // Reload groups immediatly
+        Session::loadGroups();
```

Comment:
This should be done only if session has been initialized.

Reply:
Done.